app/src/main/java/com/rafaelmfer/nasaexperience/ui/fragments/FragmentLogin.kt

```diff
@@ -59,7 +84,22 @@ class FragmentLogin : Fragment() {
                 }
             }
         }
-        sign_up.setOnClickListener { activity?.startFragment(FragmentRegister()) }
+
+        sign_up.setOnClickListener { activityContract.startFragment(FragmentRegister()) }
+
+        viewModel.loginResponse.observe(activityContract as LifecycleOwner, Observer {
+            if (it) {
+                val intent = Intent(context, ActivityHome::class.java)
+                startActivity(intent)
+            } else {
+                Toast.makeText(context, "erro ao realizar o login", Toast.LENGTH_LONG).show()
+            }
+        })
+        btGoogleSignIn = view.findViewById(R.id.btGoogleSignIn)
```

Comment:
acredito que essa linha é desnecessaria afinal estamos em kotlin, e o botao possuem o mesmo nome

Reply:
" lateinit property btGoogleSignIn has not been initialized"
Aparentemente precisamos dela para iniciar o botão. então estou mantendo.

Reply:
entendi.. ok